AsyncAwait.Tasks/AsyncAwait.Task2.CodeReviewChallenge/Controllers/HomeController.cs

```diff
@@ -27,13 +28,13 @@ public ActionResult Index()
 
     public ActionResult Privacy()
     {
-        ViewBag.Message = _privacyDataService.GetPrivacyDataAsync().Result;
+        ViewBag.Message = Task.Run(() => _privacyDataService.GetPrivacyDataAsync()).Result;

```

Comment:
This is not correct usage of async programming, please read the following https://learn.microsoft.com/en-us/dotnet/csharp/asynchronous-programming/async-scenarios#important-info-and-advice

Reply:
method should be made asynchronous, and this line should be `ViewBag.Message = await _privacyDataService.GetPrivacyDataAsync();`